src/index.ts

```
@@ -165,31 +651,140 @@ export default class Fingrprint {
+    ERROR: 'error',
+} as const;
+
+/**
+ * Manages shard ID assignments for Redis Cluster nodes.
+ */
+class ClusterShardManager extends EventEmitter {
+    #clusterShardIds: Map<string, number> = new Map();
+
+    constructor() {
+        super();
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)